setup.py

```diff
@@ -21,16 +21,15 @@
         'Programming Language :: Python :: 3',
     ],
     keywords='python3 ping icmp socket tool',
-    packages=setuptools.find_packages(exclude=['contrib', 'docs', 'tests']),
-    py_modules=["ping3", "command_line_ping3", "errors", "enums"],
+    packages=["ping3"],
     python_requires='>=3',
-    install_requires=[],
+    install_requires=["anyio >= 3"],
```

Comment:
why ping3 need `anyio` as `install_requires`?

Reply:
Oops, sorry, that is a misplaced commit from my work to anyio-ize ping3. Will fix shortly.